pkg/games/pong/src/Pong.ts

```diff
@@ -1,8 +1,9 @@
 import { Entity, Game, Renderer, System } from "@piggo-legends/gamertc";
 import { Container, Text, Texture, Sprite, Assets, Spritesheet, Graphics } from "pixi.js";
 
-const stoneTexture = Texture.from('stone.png');
-const dirtTexture = Texture.from('dirt.png');
+const floorTextureFetch = Texture.fromURL('floor_small.png');
+const stoneTextureFetch = Texture.fromURL('stone.png');
```

Comment:
did this so that it can be properly awaited. using `Texture.from`, you aren't given a promise, but the texture might not be loaded yet. was running into an issue where the texture's dimensions weren't available yet and no easy way to wait for it to be available